<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPanel</class>
 <widget class="QDialog" name="AdminPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Panel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Baseball Stadium Administrator Panel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="stadiumTab">
      <attribute name="title">
       <string>Stadium Management</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="importStadiumButton">
         <property name="text">
          <string>Import Stadium Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="stadiumTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="souvenirTab">
      <attribute name="title">
       <string>Souvenir Management</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Select Team:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="teamComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="souvenirTable"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="addSouvenirButton">
           <property name="text">
            <string>Add Souvenir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editSouvenirButton">
           <property name="text">
            <string>Edit Selected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteSouvenirButton">
           <property name="text">
            <string>Delete Selected</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 